docs/src/components/ComponentPlayground/ComponentPlaygroundTemplate.tsx

```diff
@@ -0,0 +1,67 @@
+import { KnobInspector, unstable_KnobContext } from '@stardust-ui/docs-components'
+import { Grid, Header, Segment } from '@stardust-ui/react'
+import * as _ from 'lodash'
+import * as React from 'react'
+
+import ComponentPlaygroundSnippet from './ComponentPlaygroundSnippet'
+
+type ComponentPlaygroundTemplateProps = {
+  element?: React.ReactElement
+  component?: React.FunctionComponent
+}
+
+const NoopKnobProvider: React.FunctionComponent = props => {
+  const knobContext = React.useContext(unstable_KnobContext)
+  const noopContext = { ...knobContext, registerKnob: _.noop, unregisterKnob: _.noop }
+
+  return (
+    <unstable_KnobContext.Provider value={noopContext}>
+      {props.children}
+    </unstable_KnobContext.Provider>
+  )
+}
+
+const ComponentPlaygroundTemplate: React.FunctionComponent<
+  ComponentPlaygroundTemplateProps
+> = props => {
+  // @ts-ignore
+  const shouldStretch = props.element && props.element.type.displayName === 'Divider'
+
+  return (
+    <Grid columns="75% 25%" rows="1fr auto" styles={{ gridColumnGap: '1rem' }}>
+      <Segment
```

Comment:
Why do we use Segment here?

Reply:
Why not? 🤔 It has a proper look to be used there...